src/bulletin/bulletin.controller.ts

```diff
@@ -81,4 +84,10 @@ export class BulletinController {
       }
     }
   }
+
+  @Delete(':id')
+  @UseGuards(AuthGuard)
+  async deleteBulletin(@Param('id', ParseIntPipe) id: number) {
+    return this.bulletinService.deleteBulletin(+id);
```

Comment:
Is the +id necessary since theres already a ParseIntPipe na? Please ignore if it is.